package.json

```diff
@@ -44,7 +44,7 @@
   },
   "scripts": {
     "prestart": "yarn localize && d2-manifest package.json manifest.webapp",
-    "start": "BROWSER=false PORT=${PORT:-8081} react-scripts start",
+    "start": "BROWSER=false react-scripts start",
```

Comment:
Does it make sense to move BROWSER also to env?

Reply:
Sure, I believe all env variables should be in ```.env*``` files to be honest